scripts/firedrake-install

```diff
@@ -228,16 +229,16 @@ honoured.""",
     parser.add_argument("--verbose", "-v", action="store_true", help="Produce more verbose debugging output.")
     parser.add_argument("--mpicc", type=str,
                         action="store", default=None,
-                        help="C compiler to use when building with MPI (default is 'mpicc')")
+                        help="C compiler to use when building with MPI. If not set, MPICH will be downloaded and used.")
```

Comment:
`firedrake-install` with no options should do the right thing on a workstation. In almost all cases, the only reason why you would really like to use system MPI is because you are on a cluster, which is the atypical case.

Reply:
OK.